types/try-to-catch/try-to-catch-tests.ts

```diff
@@ -0,0 +1,9 @@
+import tryToCatch = require("try-to-catch");
+
+const main = async () => {
+    await tryToCatch(() => 5); // $ExpectType [Error] | [null, number]
+
+    await tryToCatch(async () => 5); // $ExpectType [Error] | [null, number]
+
+    await tryToCatch(() => { throw new Error("error") });  // $ExpectType [Error] | [null, unknown]
```

Comment:
I'm guessing this is the missing semicolon that the tests are complaining about now.

```suggestion
    await tryToCatch(() => { throw new Error("error"); });  // $ExpectType [Error] | [null, unknown]
```